core/pom.xml

```diff
@@ -17,4 +17,12 @@
         <module>optimization</module>
     </modules>
 
+    <dependencies>
+    <dependency>
+        <groupId>com.fasterxml.jackson.datatype</groupId>
+        <artifactId>jackson-datatype-guava</artifactId>
+        <version>${jackson.version}</version>
+    </dependency>
+    </dependencies>
```

Comment:
This dependency is not at the right place (meta-module). Please move it into the "concrete" module that needs it